### ISSUE TITLE
Timeline in `Invite Attendees` is collapsed

### ISSUE THREAD

**Issue by advancingu** [ExchangeCalendar/exchangecalendar#14]
In the `Invite Attendees` dialog, the timeline widget does not expand, making it impossible to read:
![screenshot from 2017-06-07 21-12-18](https://user-images.githubusercontent.com/130207/26896927-e4488584-4bc6-11e7-970a-a1157ed47cb8.png)

**Comment by Trim** [comment]
That's quite strange. Could you please share with me the releases of:
* Operating System (seems to be Ubuntu, but which one, 17.04?)
* Thunderbird version: release 52.11.1, beta 54.0.2 ?
* Lightning release
* exchangecalendar version and from where

That will help me to reproduce bug in my environment.

**Comment by advancingu** [comment]
* Thunderbird 52.1.1 64bit
* Ubuntu 17.04 vanilla, default Unity desktop
* Lightning 5.4
* exchangecalendar v4.0.0-beta1 release file [available here](https://github.com/ExchangeCalendar/exchangecalendar/releases/tag/v4.0.0-beta1)

Thanks for looking into it!

**Comment by advancingu** [comment]
@Trim Ok, so I've debugged this a bit and found that it is caused by the `flex="1"` attribute on the first `xul:listbox` node under the `freebusy-timebar`. Once I delete the flex attribute and add CSS such as `width: 100%` on the listbox, the box expands as expected.

**Comment by Trim** [comment]
Oh sorry, I forgot to comment here. I have tested on two different computer (one physical and one virutal machine) with fresh Ubuntu 17.04 installation with exchangecalendar 4.0beta2 and I didn't see this issue.

Maybe the issue come from the Ubuntu theme after upgrade ?

According to the [flex XUL attribute](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/flex), it should already take all available space:

> Flexible elements grow and shrink to fit their given space. Elements with larger flex values will be made larger than elements with lower flex values, at the ratio determined by the two elements.

To inspect the state of other elements in the dialog, you can use the [DOM Inspector](https://addons.mozilla.org/en-US/thunderbird/addon/dom-inspector-6622/) addon which even allows you to modify in live these attributes.

**Comment by advancingu** [comment]
Yes, I debugged this in the DOM Inspector. As far as I could tell, the element only expanded when removing the `flex` attribute and adding width via CSS.
Also, can you tell if this is even an element from exchangecalendar? I couldn't find its definition in the code here, so it seems that this is from Lightning and that they need to fix this bug.

**Comment by Trim** [comment]
I've checked the code and we have an overlay on the calendar-event-attendees dialog.

However, this overlay only add some javascript code and a XUL binding.

Finally, commit 0d1a78a0e4ac40bd02e5e65ec4536cfe4940fe78 has removed the javascript code and now the overlay is still here but not usable.

I'll do a final clean to remove correctly the overlay as it seems to not be used any more.

**Comment by advancingu** [comment]
Not sure if I understand your comment. Does this mean you know how to fix the issue?

**Comment by Trim** [comment]
I wanted to say that `freebusy-timebar` is not modified by our code.

Although our code try to load some javascript file which were removed by the above mentioned commit.

We should clean up that, but I think that's not related to this issue.

**Comment by advancingu** [comment]
So this element is from Lightning?
